src/styles/_catalog.scss

```diff
@@ -1,9 +1,13 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @each $index, $width in $screen {
+    @media (min-width: $width) {
+      grid-template-columns: repeat($index, $card-width);
+    }
+  }
```

Comment:
https://mateacademy.slack.com/archives/C0506RS8F62/p1683027272658179

Reply:
agree on that